intg-denonavr/setup_flow.py

```diff
@@ -226,6 +226,16 @@ async def handle_configuration_mode(msg: UserDataResponse) -> RequestUserInput |
                 },
                 "field": {"checkbox": {"value": True}},
             },
+            {
+                "id": "volume_step",
+                "label": {
+                    "en": "Volume step",
+                    "fr": "Pallier de volume",
+                },
+                "field": {
+                    "number": {"value": 0.5, "min": 0.1, "max": 10, "steps": 0.1, "decimals": 1, "unit": {"en": "dB"}}
```

Comment:
My AVR-X2700 can't handle `"min": 0.1` / `"steps": 0.1`. Increasing by `0.1` just stays at the same volume. It seems that it only supports `0.5` and full number steps like 1, 2, .. , 10
- `0.8` is rounded up to `1.0`.
- `1.5` is rounded down to `1.0`

Does your AVR behave differently?
I'd like to set `min` to `0.5` to avoid any issues with "non-working" volume control.